### ISSUE TITLE
Proposal: Protocols and registered reports

### ISSUE THREAD

**Issue by nealhaddaway** [mcguinlu/ESinR#12]
### Chapter/Section contents 

An introductory chapter (perhaps part of an intro section) focusing on designing and publishing protocols or registered reports - explanation of benefits along with best practices.

### List of R packages that should be covered in this Chapter/Section?

markdown
bookdown

**Comment by mcguinlu** [comment]
Thanks for suggesting this @nealhaddaway 🚀

I'm really keen to cover `markdown`/`bookdown` in detail, but I think it should be in it's own chapter (something along the lines of "Literate programming for evidence synthesis") which would then cover RMarkdown for creating all sorts of ES outputs, including Protocols and reg reports. 

How does this sound to you? And were there any specific considerations related to writing reg reports/protocols in RMarkdown that you thought would be useful to cover?

**Comment by nealhaddaway** [comment]
I like your idea. I think it would be good to have an intro section on principles of good evidence synthesis that don't focus on practicalities of R packages but rather core principles of how to do ES well - mostly consisting of signposting ot external resources like the CEE Guidelines/Cochrane Handbook etc... What do you think?
________________________________

**Comment by mcguinlu** [comment]
Yeah, I definitely think we want to make sure people are well equipped with the theortical knowledge/pitfalls surrounding ES exercises before they jump into practial issues and start applying functions!
